code/modules/power/supermatter/supermatter.dm

```diff
@@ -6,9 +6,11 @@
 #define OXYGEN_HEAT_PENALTY 1
 #define CO2_HEAT_PENALTY 0.1
 #define NITROGEN_HEAT_MODIFIER -1.5
+#define HYDROGEN_HEAT_PENALTY 30
```

Comment:
H2 shouldn't have a higher heat factor than plasma (let alone double the heat factor), it doesn't really make sense considering plasma's regarded physical properties. Maybe consider further nerfing plasma in parallel or reducing the overall nerf to hydrogen.